client/gulpfile.js

```diff
@@ -57,6 +60,21 @@ const sources = {
   }
 };
 
+gulp.task('scss:compileJsToScss', () => {
+  const variablesConfigPath = 'scss/utilities/variables_config';
+  const files = fs.readdirSync(variablesConfigPath);
+
+  return files.filter(file => path.extname(file) === '.js')
+    .forEach((file) => {
+      const fileName = path.basename(file, '.js');
+      const scssFileName = `_${fileName.replace(/-/g, '_')}.scss`;
+      const fileExport = require(path.join(__dirname, variablesConfigPath, file));
+      const scssString = `$${fileName}: ${jsToSassString(fileExport)};\n`;
+
+      fs.writeFileSync(path.join(variablesConfigPath, '../compiled_variables', scssFileName), scssString);
```

Comment:
Was this because of what we spoke about?
There must be a way to use async and grunt, but bikeshed it until it's a problem.